install/kubernetes/cilium/templates/clustermesh-apiserver/deployment.yaml

```diff
@@ -208,7 +208,7 @@ spec:
         readinessProbe:
           httpGet:
             path: /readyz
-            port: apiserv-health
+            port: 9881
```

Comment:
should this be a templated helm value? (Yes it should :-) )

Reply:
This shouldn't have been included in this commit. The following overwrites this, i'll fix it for history